src/ow1/heroes/baptiste.opy

```diff
@@ -1,9 +1,13 @@
 #!mainFile "../../main.opy"
 
-playervar regen_burst_cooldown
-playervar regen_burst_hps
+# effect to show players that received the HoT
+#!define playRegenBurstHoTEffect(h) playEffect(getAllPlayers(), DynamicEffect.GOOD_EXPLOSION, Color.YELLOW, h, 1)
+
 playervar regen_burst_targets # stores players affected by Baptiste's regen burst.
+playervar exo_boots_charge_normalized # value from 0-100
+playervar exo_boots_charge_hud_visible
 
+# TODO: the progress bar initialized here is initialized twice and shows 2 bars in game. I can't figure out why
```

Comment:
@MaxwellJung do you have any idea as to why this happens?
![image](https://github.com/MaxwellJung/ow1_emulator/assets/42985687/667b2706-22ac-4f48-a322-1ebca7cfe33d)

Reply:
still don't know why it calls initialize twice, but I got it to only display one bar at a time.